pkg/client/fake/client.go

```diff
@@ -1252,8 +1252,8 @@ func inTreeResourcesWithStatus() []schema.GroupVersionKind {
 
 		{Group: "apiextensions.k8s.io", Version: "v1", Kind: "CustomResourceDefinition"},
 
-		{Group: "flowcontrol.apiserver.k8s.io", Version: "v1beta2", Kind: "FlowSchema"},
-		{Group: "flowcontrol.apiserver.k8s.io", Version: "v1beta2", Kind: "PriorityLevelConfiguration"},
+		{Group: "flowcontrol.apiserver.k8s.io", Version: "v1", Kind: "FlowSchema"},
```

Comment:
I think we should have both. This config is just for the status subresource. Most of our users will not be using kube 1.29 right away and so they might be forced to use the older version. I don't see a reason to break that.

Reply:
That's fair, I'll update it with the v1beta2 as well.